### PR TITLE
Added Some Animation to the Error Page

### LINKED ISSUE #1: Add a 404 Page
## Motivation

A 404 page is needed for unavailable routes. Gatsby ensures that your 404 page is built as 404.html as many static hosting platforms default to using this as your 404 error page. If you’re hosting your site another way you’ll need to set up a custom rule to serve this file for 404 errors.

## Modification
Add `404.js` file to `/src/pages` which would render to `404.html` and display when a user hits an unavailable route.

### PR DESCRIPTION
**Pull request by chinmaym07**
I have made the changes .
Please Review. Finally It looks like below.
Peek 2020-11-11 13-23

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -67,30 +75,60 @@ function Page404() {
...
         className={classes.backBtn}
         href="/"
+        initial={{y:'300px'}}
+        animate={{y:0}}
+        transition={{type:'spring',duration:2}}
+        whileHover={{
+            scale:1.1,
+            textShadow:`0px 0px 16px #ff4081`,
+            
+            }}
```
Comment (viveknigam3003) [inline src/pages/404.js]: The navigation back to Homepage `/` seems to be broken :/ 
Please Fix that

**Review by viveknigam3003 — CHANGES_REQUESTED** [review]
* I also see some error with MUI Typography elements. We need to look into it too.

Screenshot (291)

* Also there is a warning with `no-unused-vars`.

Screenshot (292)

* Make everything have a motion from one direction, preferably bottom-to-top, instead of "404" sliding in from the right side of the screen

* Another thing to look at is the Responsiveness of the `404.js` page when you're making any changes. Use Chrome's Dev tools to check out the view on major devices.

__Make sure the code is clean and free of unnecessary warnings and compile time errors__ 😅

**Comment by chinmaym07** [comment]
Thanks a lot for your valuable feedback .☺️
Will rectify these error in sometime..👍🏻

**Comment by chinmaym07** [comment]
I have made the changes ..
Please Review it again. :+1:

**Review by viveknigam3003 — APPROVED** [review]
Great Work @chinmaym07 !! Looks good to me! 👍🥇 
Will merge this PR! I'll be looking forward for more contributions from your side 😃

**Comment by chinmaym07** [comment]
Yeah sure. @viveknigam3003 
Thanks a lot for your Review. :grinning:  :+1:

**Pull request merged**